trunk/Admin/css/dynamic-conditions-admin.css

```diff
@@ -2,11 +2,3 @@
  * All of the CSS for your admin-specific functionality should be
  * included in this file.
  */
-
```

Comment:
Should this file still be included? It is practicaly empty at this point in time

Reply:
Right, if we need it in future, we can readd it. So it is removed now